frontend/src/__tests__/cypress/cypress/tests/mocked/modelServing/modelServingDeploy.cy.ts

```diff
@@ -101,4 +103,79 @@ describe('Model Serving Deploy Wizard', () => {
     cy.findByRole('button', { name: 'Cancel' }).click();
     cy.url().should('include', '/projects/test-project');
   });
+
+  it('Create a new deployment and submit', () => {
```

Comment:
We should consider how we could organize our mock tests assuming we get more serving platforms in the future. For now this is pretty kserve specific which can be our default but if there's platform specific things maybe we do it in a different file or create a folder structure to reflect serving backend